packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -431,6 +431,41 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
         );
         expect(logs.length).to.be.greaterThan(2);
       });
+
+      it('should return empty logs if address = ZeroAddress', async () => {
+        const logs = await relay.call(
+          RelayCalls.ETH_ENDPOINTS.ETH_GET_LOGS,
+          [
+            {
+              fromBlock: '0x0',
+              toBlock: 'latest',
+              address: ethers.ZeroAddress,
+            },
+          ],
+          requestId,
+        );
+        expect(logs.length).to.eq(0);
+      });
+
+      it('should return only logs of non-zero addresses', async () => {
+        const currentBlock = Number(await relay.call(RelayCalls.ETH_ENDPOINTS.ETH_BLOCK_NUMBER, [], requestId));
+        let blocksBehindLatest = 0;
+        if (currentBlock > 10) {
+          blocksBehindLatest = currentBlock - 10;
+        }
+        const logs = await relay.call(
+          RelayCalls.ETH_ENDPOINTS.ETH_GET_LOGS,
+          [
+            {
+              fromBlock: numberTo0x(blocksBehindLatest),
+              toBlock: 'latest',
+              address: [ethers.ZeroAddress, contractAddress2],
+            },
+          ],
+          requestId,
+        );
+        expect(logs.length).to.eq(1);
+      });
```

Comment:
Instead of duplicating code, would it not make more sense to extract the relay call into its own function and call it with params?
```
async function getLogs(params: {
  fromBlock: string;
  toBlock: string;
  address: string[];
  requestId: string;
}): Promise<any[]> {
  return await relay.call(
    RelayCalls.ETH_ENDPOINTS.ETH_GET_LOGS,
    [
      {
        fromBlock: params.fromBlock,
        toBlock: params.toBlock,
        address: params.address,
      },
    ],
    params.requestId
  );
}
```

```
const ethLogs = await getLogs({
  fromBlock: '0x0',
  toBlock: 'latest',
  address: [ethers.ZeroAddress],
  requestId,
});

const nonZeroLogs = await getLogs({
  fromBlock: numberTo0x(blocksBehindLatest),
  toBlock: 'latest',
  address: [ethers.ZeroAddress, contractAddress2],
  requestId,
});
```

Reply:
Great idea! But currently all the test units in eth_getLogs suite are using .call() method from the relay, and adding a shared method will result in code changes to the rest of other test units. Extracting the code to a shared component to avoid code duplication is always a good practice, but I believe it would be out of scope for this PR.